torch/_inductor/cudagraph_trees.py

```diff
@@ -1777,7 +1809,7 @@ def shutdown(self):
             node.graph = None
 
         self.graph = None
-        self.roots = None
+        self.roots = None  # type: ignore[assignment]
```

Comment:
I don't want to, because there's a ton of tests all over that don't test None, and I think the idea is the lifecycle roots only turns None when everything gets disarmed